vendor/engines/ldap_authentication/README.md

```diff
@@ -0,0 +1 @@
+# LdapAuthentication
```

Comment:
Can we link to the howto_ldap and provide some context on what this is used for? (Should the LDAP how-to be in this engine and not loose in open? I don't have strong opinions but that might make sense with encapsulation.)